ui/page/components/account_selector.go

```diff
@@ -200,34 +203,44 @@ func (as *AccountSelector) Layout(gtx layout.Context) layout.Dimensions {
 }
 
 func (as *AccountSelector) ListenForTxNotifications(ctx context.Context) {
+	if as.TxAndBlockNotificationListener == nil {
+		as.TxAndBlockNotificationListener = listeners.NewTxAndBlockNotificationListener()
+	}
```

Comment:
To prevent duplicate listeners being sent, this should abort/return `if as.TxAndBlockNotificationListener != nil`. This means that when this listening is done, also set `as.TxAndBlockNotificationListener` to nil after closing the channel.